### PR TITLE
メール送信を非同期にするように修正しました

### PR DESCRIPTION
**Pull request by SekiMoemi**

### CODE REVIEW

File: src/main/java/jp/co/example/ecommerce_a/service/OrderService.java [hunk 1]
```diff
@@ -33,6 +34,7 @@ public class OrderService {
 	 * @param form 注文時に使用するフォーム
 	 * @throws ParseException
 	 */
+	@Async
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/service/OrderService.java:37]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/service/OrderService.java:37]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 8, expected level should be 2.

File: src/main/java/jp/co/example/ecommerce_a/common/AsyncConfig.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package jp.co.example.ecommerce_a.common;
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/AsyncConfig.java:1]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.PackageNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Package name 'jp.co.example.ecommerce_a.common' must match pattern '^[a-z]+(\.[a-z][a-z0-9]*)*$'.

File: src/main/java/jp/co/example/ecommerce_a/common/AsyncConfig.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.task.AsyncTaskExecutor;
+import org.springframework.scheduling.annotation.EnableAsync;
+import org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor;
+
+@Configuration
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/AsyncConfig.java:9]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.MissingJavadocTypeCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing a Javadoc comment.

File: src/main/java/jp/co/example/ecommerce_a/common/AsyncConfig.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+@EnableAsync
+public class AsyncConfig {
+    @Bean
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/AsyncConfig.java:12]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 4, expected level should be 2.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/AsyncConfig.java:12]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.MissingJavadocMethodCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing a Javadoc comment.

File: src/main/java/jp/co/example/ecommerce_a/common/AsyncConfig.java [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    public AsyncTaskExecutor taskExecutor() {
+        ThreadPoolTaskExecutor executor = new ThreadPoolTaskExecutor();
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/AsyncConfig.java:14]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 8, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/common/AsyncConfig.java [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+        executor.setCorePoolSize(10);
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/AsyncConfig.java:15]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 8, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/common/AsyncConfig.java [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+        executor.setMaxPoolSize(10);
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/AsyncConfig.java:16]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 8, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/common/AsyncConfig.java [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+        executor.setQueueCapacity(10);
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/AsyncConfig.java:17]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 8, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/common/AsyncConfig.java [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+        return executor;
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/AsyncConfig.java:18]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 8, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/common/AsyncConfig.java [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+    }
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/AsyncConfig.java:19]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def rcurly' has incorrect indentation level 4, expected level should be 2.

**Review by yamada-kensyu — APPROVED** [review]
承認します！

**Pull request merged**